src/components/graph/graph.config.js

```diff
@@ -303,5 +303,8 @@ export default {
     markerHeight: 6,
     markerWidth: 6,
     type: "STRAIGHT",
+    strokeDasharray: 0,
```

Comment:
Please add the @jsdoc for the new properties: `strokeDasharray`, `strokeDashoffset` and `strokeLinecap` with the correct default values. Thanks!